management/prompt_test.go

```diff
@@ -293,6 +332,35 @@ func givenAPartialPrompt(t *testing.T, prompt PromptType) *PromptScreenPartials
 	return partials
 }
 
+func givenAPromptRendering(t *testing.T) *PromptRendering {
+	t.Helper()
+
+	settings := &PromptRendering{
+		RenderingMode:           auth0.String("advanced"),
+		ContextConfiguration:    &[]string{"branding.settings", "branding.themes.default"},
+		DefaultHeadTagsDisabled: auth0.Bool(false),
+		HeadTags: []interface{}{
+			map[string]interface{}{
+				"tag":     "script",
+				"content": "",
+				"attributes": map[string]interface{}{
+					"defer": true,
+					"src":   "https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js",
+					"async": true,
+					"integrity": []string{
+						"sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==",
+					},
+				},
+			},
+		},
+	}
+
+	err := api.Prompt.UpdateRendering(context.Background(), PromptSignup, ScreenSignup, settings)
+	assert.NoError(t, err)
```

Comment:
I think the API calling is already in the unit test : https://github.com/auth0/go-auth0/pull/458/files#diff-b7e2d3271a58d0bd6cb38b3ea977640eefc43fd56f013d7c4c2d7b226f7f8474R156